github/github-actions-ci/includes/2-github-actions-workflows-ci.md

```diff
@@ -177,6 +195,18 @@ Another action we could take is to add a label to the pull request. In this case
          ADD_LABEL: "approved"
 ```
 
-Notice the block called ```env:```. This is where you set the environment variables for this action. For example, you can set the number of approvers needed. Here, it's one. The ```GITHUB_TOKEN``` variable is required because the action must make changes to your repository by adding a label. Finally, you supply the name of the label to add.
+Notice the block called ```env:```. This is where you set the environment variables for this action. For example, you can set the number of approvers needed. Here, it's one. The ```GITHUB_TOKEN``` variable is required because the action must make changes to your repository by adding a label. GitHub Actions lets you control the permissions granted to the GITHUB_TOKEN secret. A `permissions` key supported at the workflow and job level enables you to specify which permissions you want for the token. Any permission that is absent from the list will be set to `none`. Finally, you supply the name of the label to add.
+
+GitHub actions now requires a collaborator with write access to take action to run workflows from first time users in pull requests.
```

Comment:
@rmallorybpc I don't think this line fits in this section. I think it would be better to include this information in a section that talks about running workflows instead of automating reviews.

Reply:
@a-a-ron ok, I added the below section in [Build and deploy applications to Azure by using GitHub Actions](https://github.com/githubpartners/microsoft-learn/pull/6/files):
GitHub Actions lets you control the permissions granted to the `GITHUB_TOKEN` secret. The `GITHUB_TOKEN` is an automatically generated secret letting you make authenticated calls to the GitHub API in your workflow runs. Actions generates a new token for each job and expires the token when a job completes. These new settings allow you to follow a principle of least privilege in your workflows. The token has `write` permissions to a number of API endpoints except in the case of pull requests from forks which are always `read`.

Reply:
I spent some time looking into this and I think we can add a new "short" section to add clarity around this statement. I really like the blog post about it: https://github.blog/2021-04-22-github-actions-update-helping-maintainers-combat-bad-actors/

Specifically the second feature in this section: https://github.blog/2021-04-22-github-actions-update-helping-maintainers-combat-bad-actors/#new-features-to-help-protect-maintainers

We don't really have an actions admin module that this would fit into other than admin for actions enterprise, but I don't really want to put it there since it's not enterprise specific. For now, I'm thinking here: https://docs.microsoft.com/en-us/learn/modules/github-actions-automate-tasks/2c-configure-github-actions-workflow

Reply:
Good morning @a-a-ron I added content, created a new branch, and tagged you for review.